plugin/Categorifier/Categorize.hs

```diff
@@ -1,11 +1,11 @@
 {-# LANGUAGE ExistentialQuantification #-}
 {-# LANGUAGE TemplateHaskell #-}
 
--- | These are the operations to use with the "Kitty.Plugin" plugin, which trigger conversion from
+-- | These are the operations to use with the "Categorifier" plugin, which trigger conversion from
 --  `(->)` to some target category.
 --
 --  __NB__: This module is expected to be imported qualified.
-module Kitty.Plugin.Categorize
+module Categorifier.Categorize
```

Comment:
Should this be `Categorifier.Categorify`?

Reply:
Yeah, I was going to do at least a separate commit. This one was mostly to get rid of the various meanings of “kitty”. But I’m happy for both to be in the same PR.